awx/main/models/workflow.py

```diff
@@ -186,19 +127,29 @@ def create_wfjt_node_copy(self, user, workflow_job_template=None):
         is not allowed to access
         '''
         create_kwargs = {}
+        allowed_creds = []
         for field_name in self._get_workflow_job_field_names():
+            if field_name == 'credentials':
+                Credential = self._meta.get_field('credentials').related_model
```

Comment:
What's the need to look up the class this way?  Something to make migrations happy?

Reply:
Right, I do this in several places. It's because of the import order. If unified_jobs.py imports Credential and credential.py imports UnifiedJob, then you get a startup server error. This was most important for the UnifiedJob <-> Schedule relationships. But here I'm just trying to avoid putting credentials higher up the chain the workflows. Granted, we could probably safely import Credential here safely, because workflow should be unlikely to be imported by other models (it's kind of the last one down the chain).